### DISCUSSION TITLE
How to load appsettings.json?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#2388] [Q&A]
I have `appsettings.json` in the Blazor WASM client `wwwroot` folder. It's set to `Build Action = Content` and `Copy to Output Director = Do not copy`. I can't get it to load any of my settings. What am I missing?

**Comment by ElderJames** [comment]
You may need to inject the `IConfiguration`.

https://docs.microsoft.com/en-us/aspnet/core/blazor/fundamentals/configuration?view=aspnetcore-6.0

**Comment by michaelcsikos** [comment]
@ElderJames How do we get `appsettings.json` to work in Blazor Server? Is there an example?

`EDIT` The instructions are here: https://github.com/ant-design-blazor/blazor-pro-components
